src/os/os_chmod_test.go

```diff
@@ -28,4 +30,26 @@ func TestChmod(t *testing.T) {
 		t.Fatalf("chmod %s %#o: %s", f.Name(), fm, err)
 	}
 	checkMode(t, f.Name(), fm)
+
+}
+
+func TestChown(t *testing.T) {
+	f := newFile("TestChown", t)
+	defer Remove(f.Name())
+	defer f.Close()
+
+	if runtime.GOOS != "windows" {
+		if err := Chown(f.Name(), 0, 0); err != nil {
+			t.Fatalf("chown %s 0 0: %s", f.Name(), err)
```

Comment:
So in terms of errors like this, and maybe this project is different, but the pattern is either
Chown(%s, 0, 0): %v != nil
or 
Chown(%s, 0, 0): got %v, want nil
It's just so you have a very regular pattern for when you need to deal with logs.
This was pretty much de rigeur at google.